frontends/web/src/components/rates/rates.tsx

```diff
@@ -113,7 +113,7 @@ function Conversion({
   children,
 }: PropsWithChildren<Props>): JSX.Element | null {
 
-  let formattedValue = '---';
+  let formattedValue = ' N/A';
```

Comment:
`+ N/A` and `- N/A` are not much better :innocent: 

I think the `---` is not the problem (both `---` and `N/A` are okay`), but that there is a sign in `+` or `-` in front, coming from https://github.com/digitalbitbox/bitbox-wallet-app/blob/master/frontends/web/src/components/transactions/transaction.tsx#L207. 

Not sure why this is passed as `children` anyway. Maybe a good opportunity to make it a property instead, and not render it if there is no amount to render.

Reply:
ok, pushed an update to improve it